src/MLJ.jl

```diff
@@ -119,7 +119,6 @@ MMI.metadata_model(
     target_scitype = AbstractVector{<:MMI.Continuous},
     weights = false,
     path = "EvoTrees.EvoTreeRegressor",
-    descr = EvoTreeRegressor_desc,
```

Comment:
I guess we can remove the short descriptions in lines 97-101 as well?

Reply:
Good catch. Done.